internal/terraform/datadir/module_types.go

```diff
@@ -16,44 +17,37 @@ const (
 	GIT        ModuleType = "git"
 )
 
-var moduleSourceLocalPrefixes = []string{
-	"./",
-	"../",
-	".\\",
-	"..\\",
-}
-
 // GetModuleType parses source addresses to determine what kind of source the Terraform module comes
 // from. It currently supports detecting Terraform Registry modules, GitHub modules, Git modules, and
 // local file paths
-func (r *ModuleRecord) GetModuleType() ModuleType {
+func GetModuleType(sourceAddr string) ModuleType {
```

Comment:
Is it worth moving this and dependent logic out of the `datadir` package, given that it no longer deals with any data within `.terraform` (the data dir)? Given that `module.calls` command/handler is the only consumer it might make sense to move it to the same file, where we already have `getModuleDocumentationLink()`?